jerry-debugger/jerry-client-ws.html

```diff
@@ -671,6 +738,30 @@
         return;
       }
 
+      case JERRY_DEBUGGER_EVAL_RESULT:
+      case JERRY_DEBUGGER_EVAL_RESULT_END:
+      case JERRY_DEBUGGER_EVAL_ERROR:
+      case JERRY_DEBUGGER_EVAL_ERROR_END:
+      {
+        evalResult = concatUint8Arrays(evalResult, message);
+
+        if (message[0] == JERRY_DEBUGGER_EVAL_RESULT_END)
+        {
+          appendLog(cesu8ToString(evalResult));
+          evalResult = null;
+          return;
+        }
+
+        if (message[0] == JERRY_DEBUGGER_EVAL_ERROR_END)
+        {
+          appendLog("ERROR: " + cesu8ToString(evalResult));
```

Comment:
Currently the error message processing looks like this:

    if (is_object(result))
      if (has_message_property(as_object(result)))
        send(as_object(result).message)
      else
        send(get_class_name(object))
    else
       send(to_string(result))

My problem was with to_string is that it can throw another error...